tutorials/math/principal.py

```diff
@@ -0,0 +1,84 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Principal Components Analysis (PCA) example
+##
+## Example of using TPrincipal as a stand alone class.
+##
+## I create n-dimensional data points, where c = trunc(n / 5) + 1
+## are  correlated with the rest n - c randomly distributed variables.
+##
+## Based on principal.C by Rene Brun and Christian Holm Christensen
+##
+## \macro_output
+## \macro_code
+##
+## \authors Juan Fernando Jaramillo Botero
+
+from ROOT import TPrincipal, gRandom, TBrowser, vector
+
+
+n = 10
+m = 10000
+
+c = n / 5 + 1
+
+print ("""*************************************************")
```

Comment:
I guess you'd want to remove the ``")`` at the end of this line.

Reply:
Done